students/jemielity_kamil/lesson_07_string_datetime/to_swap_the_two_words.py

```diff
@@ -0,0 +1,4 @@
+string = input('String: ')
+first_word = string[:string.find(' ')]
+second_word = string[string.find(' ') + 1:]
+print(second_word + " " + first_word)
```

Comment:
Please try to use format() in the next lessons.

Just for your information, instead of 'find()' you might also use 'split()'
```
first_word = string.split()[0]
next_words = " ".join(string.split()[1:])
print("{} {}".format(next_words, first_word))
```